Add CVE Explorer documentation page (#36191)
DataDog/documentation: content/en/security/code_security/software_composition_analysis/_index.md

To learn more about how the inventory is generated, how Static and Runtime data differ, and how to interpret the library details (usage, vulnerabilities, licenses, versions, and OpenSSF score), see [Library Inventory][14].

### Explore the full CVE catalog

> The [CVE Explorer][15] gives you a searchable catalog of every CVE and security advisory tracked by Datadog, regardless of whether it affects your environment. Use it to proactively assess exposure to newly published vulnerabilities before they appear in your findings.

Reviewer: ```suggestion
Use the [CVE Explorer][15] to search every CVE and security advisory tracked by Datadog, including those that do not affect your environment. This helps you assess exposure to newly published vulnerabilities before they appear in your findings.
```

> For CVEs that affect packages detected in your scanned repositories and services, Datadog automatically marks them as impacted. Assets that have not been scanned will not reflect an impacted status.

Reviewer: ```suggestion
For CVEs that affect packages detected in your scanned repositories and services, Datadog automatically marks them as impacted. Assets that have not been scanned do not show an impacted status.
```
